### DISCUSSION TITLE
Drag And Drop UI Scaling Issue

### DISCUSSION THREAD

**Discussion by williamrousseau** [unoplatform/uno#13896] [Q&A] [answered]
I'm currently implementing Drag & Drop in a ListView and a Grid and encounter an issue with the dragged component UI. It does not rescale in function of the Scale of the Monitor:

## Dragged Component UI when monitor scale is 100% (Works Fine):

![UnoPlatformDragAndDrop100](https://github.com/unoplatform/uno/assets/43794005/b3d6649d-e247-4088-aa52-9938d7195a5d)
![image](https://github.com/unoplatform/uno/assets/43794005/25640002-a860-462e-831e-b55fed8d1dc0)

## Dragged Component UI when monitor scale is 125% (Offset and wrong scale):

![UnoPlatformDragAndDrop125](https://github.com/unoplatform/uno/assets/43794005/0e70dc84-59a4-45a1-9ef6-782afa58c515)
![image](https://github.com/unoplatform/uno/assets/43794005/ddcb11da-95c1-45e5-a43c-e8e399dea4d3)

<br>
As you can see, the dragged component UI is the same with 100% scale and 125% scale, but everything else has the correct scale size.
This issue can cause an inconvenience for the user if he uses a different scale than 100% on his computer.
This issue has been tested on ListView items and Grid item, both have the issue.
The Drag & Drop has been tested in a WuiUI3 application and the rescaling works fine.

<br>
Thank you!

**Comment by jeromelaban** [comment]
Thanks for the report. 

You're mentioning that you've tested with WinUI 3, but assuming you created your test project using the Uno Platform templates, could you tell which project head you're running with for the screenshots above?

**Reply by williamrousseau** [reply]
The screenshots above are with a new project created using the latest Uno Platform App project template from Visual Studio.

**Reply by williamrousseau** [reply]
Here's a list of all packages used in Uno Platform project:

		<PackageReference Include="Uno.Resizetizer" Version="1.0.2" />
		<PackageReference Include="Uno.WinUI" Version="4.8.15" />
		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.2.230313.1" />
		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
		<PackageReference Include="CommunityToolkit.Mvvm" Version="8.1.0" />
		<PackageReference Include="Uno.Extensions.Configuration" Version="2.3.2" />
		<PackageReference Include="Uno.Extensions.Http" Version="2.3.2" />
		<PackageReference Include="Uno.Extensions.Http.Refit" Version="2.3.2" />
		<PackageReference Include="Uno.Extensions.Logging.WinUI" Version="2.3.2" />
		<PackageReference Include="Uno.Extensions.Serialization.Http" Version="2.3.2" />
		<PackageReference Include="Uno.Extensions.Serialization.Refit" Version="2.3.2" />
		<PackageReference Include="Uno.Material.WinUI" Version="2.5.3" />
		<PackageReference Include="Uno.Toolkit.WinUI.Material" Version="2.5.5" />
		<PackageReference Include="Uno.Toolkit.WinUI" Version="2.5.5" />
		<PackageReference Include="Uno.Extensions.Authentication.WinUI" Version="2.3.2" />
		<PackageReference Include="Uno.Extensions.Hosting.WinUI" Version="2.3.2" />
		<PackageReference Include="Uno.Extensions.Localization.WinUI" Version="2.3.2" />
		<PackageReference Include="Uno.Extensions.Navigation.Toolkit.WinUI" Version="2.3.2" />
		<PackageReference Include="Uno.Extensions.Navigation.WinUI" Version="2.3.2" />
		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
		<PackageReference Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
		<PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.8.15" />

**Reply by jeromelaban** [reply, marked as answer]
Thanks. It seems that you're using an older version of Uno, and likely its template. At the time, the default template was missing scaling compatibility for the WinAppSDK head, [which you may find here](https://github.com/unoplatform/uno.templates/blob/main/src/Uno.Templates/content/unoapp/MyExtensionsApp._1.Windows/app.manifest), and needs to be [setup like this](https://github.com/unoplatform/uno.templates/blob/bbb06576dbb67db98d59c44f7bc721db1230ab94/src/Uno.Templates/content/unoapp/MyExtensionsApp._1.Windows/MyExtensionsApp._1.Windows.csproj#L6).

**Reply by williamrousseau** [reply]
It fixed the issue.
Thank you!
